tools/zip2mkv.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import print_function, unicode_literals
+
+import shutil
+import sys
+import os.path
+from logparser2 import sensor_gen
+from zipfile import ZipFile
+import tempfile
+import subprocess
+
+
+def name2mtime(metapath, copy_func):
+    srcdir = os.path.dirname(metapath)
+    destdir = srcdir+'/img'
+    os.mkdir(destdir)
+    start = None
+    for ts, id, (filename, _) in sensor_gen(metapath, [b"camera"]):
+        if start is None:
+            start = ts
+        filename = os.path.basename(filename)
```

Comment:
maybe add note, how `filename` looks like? i.e. why -7 is correct cut? What will happen for long run (more than 1000 images)?